### ISSUE TITLE
[BUG] - Failing integration tests traceback

### ISSUE THREAD

**Issue by resmo** [vultr/ansible-collection-vultr#62]
```
 TASK [snapshot : setup instance] ***********************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: KeyError: 'id'
fatal: [testhost]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"<stdin>\", line 141, in <module>\n  File \"<stdin>\", line 133, in _ansiballz_main\n  File \"<stdin>\", line 81, in invoke_module\n  File \"/usr/lib/python3.10/runpy.py\", line 224, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib/python3.10/runpy.py\", line 96, in _run_module_code\n    _run_code(code, mod_globals, init_globals,\n  File \"/usr/lib/python3.10/runpy.py\", line 86, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_vultr.cloud.instance_payload_8qulosg_/ansible_vultr.cloud.instance_payload.zip/ansible_collections/vultr/cloud/plugins/modules/instance.py\", line 712, in <module>\n  File \"/tmp/ansible_vultr.cloud.instance_payload_8qulosg_/ansible_vultr.cloud.instance_payload.zip/ansible_collections/vultr/cloud/plugins/modules/instance.py\", line 708, in main\n  File \"/tmp/ansible_vultr.cloud.instance_payload_8qulosg_/ansible_vultr.cloud.instance_payload.zip/ansible_collections/vultr/cloud/plugins/module_utils/vultr_v2.py\", line 283, in present\n  File \"/tmp/ansible_vultr.cloud.instance_payload_8qulosg_/ansible_vultr.cloud.instance_payload.zip/ansible_collections/vultr/cloud/plugins/modules/instance.py\", line 606, in create_or_update\n  File \"/tmp/ansible_vultr.cloud.instance_payload_8qulosg_/ansible_vultr.cloud.instance_payload.zip/ansible_collections/vultr/cloud/plugins/module_utils/vultr_v2.py\", line 261, in wait_for_state\nKeyError: 'id'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
```

**Comment by resmo** [comment]
```
TASK [instance : test absent instance] *****************************************
fatal: [testhost]: FAILED! => {"changed": false, "fetch_url_info": {"body": "{\"error\":\"Unable to destroy server: Unable to remove VM: Server is currently locked\",\"status\":500}", "connection": "close", "content-type": "application/json", "date": "Mon, 26 Jun 2023 14:09:01 GMT", "msg": "HTTP Error 500: Internal Server Error", "server": "nginx", "status": 500, "strict-transport-security": "max-age=63072000", "transfer-encoding": "chunked", "url": "https://api.vultr.com/v2/instances/44469011-c2f4-4d81-bcdb-af51e41a9562", "x-content-type-options": "nosniff", "x-frame-options": "DENY", "x-robots-tag": "noindex,noarchive", "x-user": "ansible+opensource@vultr.com"}, "msg": "Failure while calling the Vultr API v2 with DELETE for \"/instances/44469011-c2f4-4d81-bcdb-af51e41a9562\"."}
```

**Comment by resmo** [comment]
closed by #89
